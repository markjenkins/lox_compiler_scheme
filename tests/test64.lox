{
    var i=1;
    while (i<4){
	i=i+1;
    }
    print i;
}
