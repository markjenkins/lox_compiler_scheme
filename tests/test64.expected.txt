OP_CONSTANT 1
OP_GET_LOCAL 0
OP_CONSTANT 4
OP_LESS
OP_JUMP_IF_FALSE 12
OP_POP
OP_GET_LOCAL 0
OP_CONSTANT 1
OP_ADD
OP_SET_LOCAL 0
OP_POP
OP_LOOP 20
OP_POP
OP_GET_LOCAL 0
OP_PRINT
OP_POP
OP_RETURN
